packages/rfw/lib/src/flutter/material_widgets.dart

```diff
@@ -188,6 +192,45 @@ Map<String, LocalWidgetBuilder> get _materialWidgetsDefinitions => <String, Loca
     );
   },
 
+  'DropdownButton': (BuildContext context, DataSource source) {
+    final length = source.length(['items']);
+    final dropdownMenuItems = List<DropdownMenuItem<Object>>.generate(
+        length,
+        (i) => DropdownMenuItem<Object>(
+            onTap: source.voidHandler(['items', i, 'onTap']),
+            value: source.v<String>(['items', i, 'value']),
+            enabled: source.v<bool>(['items', i, 'enabled']) ?? true,
+            alignment: ArgumentDecoders.alignment(source, ['items', i, 'alignment']) ?? AlignmentDirectional.centerStart,
+            child: source.child(['items', i, 'child'])));
```

Comment:
usual convention is to use the formatting you get when you give a trailing comma here (see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#if-you-have-a-newline-after-some-opening-punctuation-match-it-on-the-closing-punctuation)

Reply:
Thanks for the guidance. Fixed.